### DISCUSSION TITLE
Inter Display beta 2

### DISCUSSION THREAD

**Discussion by rsms** [rsms/inter#452] [Announcements]
There's a [new beta version of Inter Display available](https://github.com/rsms/inter/releases/tag/display-beta-2). The changes over beta 1 are very minor.

EDIT: Discussion moved here: https://github.com/rsms/inter/discussions/463

**Comment by ThomasRettig** [comment]
It would be nice to have a little changelog :)

**Comment by rsms** [comment]
I've begun work to integrate Display into main Inter as a variable-font axis. See #463
